src/applications/appeals/testing/components/IssueCard.jsx

```diff
@@ -27,7 +27,8 @@ export const IssueCardContent = ({
 }) => {
   // May need to throw an error to Sentry if any of these don't exist
   // A valid rated disability *can* have a rating percentage of 0%
-  const showPercentNumber = (ratingIssuePercentNumber || '') !== '';
+  const showPercentNumber =
+    (ratingIssuePercentNumber || approxDecisionDate || '') !== '';
```

Comment:
Does `showPercentNumber` always return true because `'' !== ''` is always true?

Reply:
"We want the percent to show if it's defined or if the issue has a approxDecisionDate (means it's API loaded)... the description shows either a percent or N/A"
"It probably doesn't need the ratingIssuePercentNumber part"

Reply:
Fixed